pnl-subgraph/src/utils/updateUserPositionWithSell.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/ban-types */
+
+import { Address, BigInt } from '@graphprotocol/graph-ts';
+
+import { loadOrCreateUserPosition } from './loadOrCreateUserPosition';
+
+import { COLLATERAL_SCALE } from '../../../common/constants';
+
+const updateUserPositionWithSell = (
```

Comment:
what happens if a user splits as their first action? the `adjustedAmount` is used right? then what if they sell? wouldn't both use 0 as the `adjustedAmount`